src/test/resources/com/puppycrawl/tools/checkstyle/checks/blocks/rightcurly/InputRightCurlyAloneOrSingleline.java

```diff
@@ -209,4 +209,21 @@ public void codeAfterLastRightCurly() {
         }; // violation
         for (int i = 0; i < 1; i++) { new Object(); }; // violation
     }
+
+    public void testMethod() {}; //violation
+
+    public void testMethod1() {
+    }; //violation
```

Comment:
It is not clear, how to avoid the violation for `alone or singleline`.
Please add
```
    public void testMethod1() {
    }
    ; // OK
```
Or how it should be.

Reply:
done